src/nvim/eval.c

```diff
@@ -1754,10 +1754,10 @@ ex_let_one (
           vim_setenv(name, p);
           if (STRICMP(name, "HOME") == 0)
             init_homedir();
-          else if (didset_vim && STRICMP(name, "VIM") == 0)
+          else if (didset_vim && (STRICMP(name, "VIM") == 0 || STRICMP(name, "NVIM") == 0))
```

Comment:
Why is `STRICMP` being used over `STRCMP`? I don't mean to come across as blunt, I'm just curious because of what was discussed here: https://github.com/neovim/neovim/pull/1927#discussion_r23903426